geode-core/src/main/java/org/apache/geode/internal/cache/persistence/PersistentMemberPattern.java

```diff
@@ -39,19 +39,19 @@
   protected long revokedTime;
```

Comment:
I noticed all the instance variables in the class are protected even though they can be private

Reply:
I pushed another commit that changes the fields to private and adds in missing `@Override` annotations.